### PR TITLE
Styled Desktop Version

### PR DESCRIPTION
**Pull request by FevenSeyfu**
Added feature:

- Added style for the Desktop version
- Added Responsiveness with media query
- Added style for hover effect

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,7 +28,19 @@
...
+                <ul class="nav-items">
+                    <li><a href="#" class="nav-item">Portfolio</a></li>
+                    <li><a href="#" class="nav-item">About</a></li>
+                    <li><a href="#" class="nav-item">Contact</a></li>
+                    <li class="nav-item">
+                        <a href="#">
+                            <img src="images/icons/Mail-icon.svg" alt="Mail icon" class="mail-icon">
+                        </a>
+                    </li>
+                </ul>
```
Comment (wale-prog) [inline index.html]: You have two  `nav`  items, one for the desktop screen and the other for a small screen but both of the nav items are showing on top of each other on the desktop screen. See screenshot below

image

Check my comment below for a possible fix.

File: style.css [hunk 2]
```diff
@@ -563,7 +536,451 @@ ul{
...
+        height: 100%;
+        width: 100%;
+        max-width: 100%;
+        min-width: 100%;
+    }
+    body {
+        box-sizing: border-box;
+    }
+    .desktop{
+        display: contents;
```
Comment (wale-prog) [inline style.css:488]: You could change this to `display: none;` to stop the nav items mentioned above from displaying on top of each other.

File: index.html [hunk 3]
```diff
@@ -97,222 +106,235 @@ <h2 class="supporting-text">I'am a Software Developer</h2>
...
     <section class="about-section">
-        <div>
-            <h1  class="primary-text">about me</h1>
-        </div>
-        <div>
-            <h2 class="description">
-            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </h2>
+        <div class="about-container">
+            <div>
```
Comment (wale-prog) [inline index.html:217]: The About me section overlaps the rest of the recent works section on the desktop and the recent work section images and descriptions are not showing up in mobile view. See screenshots below:

- Desktop
image

- Mobile

image

**Review by wale-prog — CHANGES_REQUESTED** [review]
## Hi @FevenSeyfu  :wave:  , 
# Status : Required Changes :recycle: 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Highlights
- No linters error :heavy_check_mark:
- Good commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (FevenSeyfu) [reply index.html]: Thank you for your feedback there were two nav items for the links that were duplicated while I was trying to fix merge conflicts. I have fixed the issue.

File: style.css (hunk 2 above)
Comment (FevenSeyfu) [reply style.css:488]: Thank you for the suggested, well noted.

File: index.html (hunk 3 above)
Comment (FevenSeyfu) [reply index.html:217]: Updated

- position of about section to resolve overlapping divs
- added content and background to mobile version work cards.

File: style.css [hunk 4]
```diff
@@ -416,27 +455,463 @@ ul{
...
+        flex-direction: column;
+        justify-content:  center;
+        gap: 24px;
+        width: 100%;
+        height: 1379px;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+        top: 1643px;
+    }
```
Comment (pinkmoon25) [inline style.css]: - [ ]Good job Implementing the changes requested by the previous reviewer, I think while making the changes there is an issue introduced, now your projects cards section are covering the first featured project please see 
image

Please fix this layout issue and you will be done :+1:

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @FevenSeyfu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights
linter checks :heavy_check_mark: 
Professional Readme :heavy_check_mark: 
Good commits messages :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (FevenSeyfu) [reply style.css]: Thank you for your feedback I have changed the position to resolve overlappings.

**Review by frank1738 — APPROVED** [review]
Hi @FevenSeyfu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**